### PR TITLE
style: readme formatting

### PR DESCRIPTION
**Pull request by MoskalykA**

### CODE REVIEW

**Review by CsokiHUN — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -20,28 +20,28 @@ If you see someone who is making an extra effort to ensure our community is welc
 
 The following behaviors are expected and requested of all community members:
 
- * Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
```
Comment (Mycroft-Studios) [inline CODE_OF_CONDUCT.md:23]: Did u just format a Github provided document? lol

**Comment by Mycroft-Studios** [comment]
this mostly just seems like formatting default documents that are provided by Github ngl. not very needed....

**Comment by MoskalykA** [comment]
> this mostly just seems like formatting default documents that are provided by Github ngl. not very needed....

The principle of open-source is to have public participation, why do you need to have it? Even if some things come from Github, they are not necessarily perfect. Then the CODE_OF_CONDUCT file is just shared by Github, at least the version you have

**Comment by MoskalykA** [comment]
I didn't delete it on purpose but finally you are right, too superficial contribution

**Pull request closed without merging**